build.rs

```diff
@@ -0,0 +1,48 @@
+use std::error::Error;
+use std::ffi::OsString;
+use std::process::{exit, Command};
+
+fn main() {
+    let rustc = std::env::var_os("RUSTC").unwrap_or_else(|| OsString::from("rustc"));
+    let command = format!("`{} --version`", rustc.to_string_lossy());
+
+    let output = Command::new(&rustc)
+        .arg("--version")
+        .output()
+        .unwrap_or_else(|e| {
+            eprintln!("Error: failed to run {}: {}", command, e);
+            exit(1)
+        });
```

Comment:
Use `rustc_version` instead of doing this manually.

Reply:
`rustc_version` has an MSRV of 1.32.0 so if we're going to bump the MSRV we might as well also bump it past 1.36.0 so this hack isn't needed at all.

Reply:
Let's just bump it to 1.36 then to keep regex happy.